<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-style:italic;&quot;&gt;Summer&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;A melhor forma de organizar sua biblioteca&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">image: url(:/newPrefix/arvore.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdicionar_livro"/>
    <addaction name="actionProcurar_livro"/>
    <addaction name="actionAdicionar_alunos"/>
    <addaction name="actionSair"/>
    <addaction name="actionAjuda"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdicionar_livro"/>
   <addaction name="actionProcurar_livro"/>
   <addaction name="actionAdicionar_alunos"/>
   <addaction name="actionAjuda"/>
   <addaction name="actionSair"/>
  </widget>
  <action name="actionAdicionar_livro">
   <property name="icon">
    <iconset>
     <normaloff>icon sis/1486485557-add-create-new-more-plus_81188.png</normaloff>icon sis/1486485557-add-create-new-more-plus_81188.png</iconset>
   </property>
   <property name="text">
    <string>Adicionar livro</string>
   </property>
  </action>
  <action name="actionProcurar_livro">
   <property name="icon">
    <iconset>
     <normaloff>icon sis/search_book_open_search_locate_6178.png</normaloff>icon sis/search_book_open_search_locate_6178.png</iconset>
   </property>
   <property name="text">
    <string>Procurar livro</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset>
     <normaloff>icon sis/stop_exit_removerelete_sto_11059.png</normaloff>icon sis/stop_exit_removerelete_sto_11059.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAdicionar_alunos">
   <property name="icon">
    <iconset>
     <normaloff>icon sis/userpassword_deusuari_787.png</normaloff>icon sis/userpassword_deusuari_787.png</iconset>
   </property>
   <property name="text">
    <string>Alterar senha</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="icon">
    <iconset>
     <normaloff>icon sis/help_question_1566.png</normaloff>icon sis/help_question_1566.png</iconset>
   </property>
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon sis/arv.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
